sql-odbc/src/odfesqlodbc/es_info.cpp

```diff
@@ -729,6 +734,15 @@ ESAPI_Tables(HSTMT hstmt, const SQLCHAR *catalog_name_sql,
         std::string query;
         GenerateTableQuery(query, flag, table_name, result_type, table_valid);
 
+        // Get list of columns with SELECT * query since columns doesn't match
```

Comment:
IIRC, this won't be applicable here since for SQLTables is using SHOW TABLES query which doesn't have column information.

Reply:
And list_of_columns isn't used in this function afterwards

Reply:
Misplaced changes; will remove

Reply:
Removed